.github/workflows/build.yml

```diff
@@ -0,0 +1,24 @@
+on: [ pull_request, push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up JDK 11 ☕
+      uses: actions/setup-java@v4
+      with:
+        java-version: 11
+        distribution: 'temurin'
+    - name: Setup Gradle 🛠️
+      uses: gradle/gradle-build-action@v3
```

Comment:
Do we need this if we use the Gradle wrapper?

Reply:
It cares about caching the actual gradle installation, otherwise it need to be downloaded each and every time.